app/packs/src/components/QuillEditor.js

```diff
@@ -71,31 +71,14 @@ export default class QuillEditor extends React.Component {
     this.initQuill();
   }
 
-  // eslint-disable-next-line camelcase
-  UNSAFE_componentWillReceiveProps(nextProps) {
-    const nextVal = nextProps.value;
-    const oldContents = this.editor ? this.getContents() : null;
-
-    if (oldContents && nextVal &&
-      !_.isEqual(nextVal.ops, oldContents.ops)) {
-      this.setState({ value: nextVal });
-      const sel = this.editor.getSelection();
-      this.editor.setContents(nextVal);
-      if (sel) this.editor.setSelection(sel);
-    }
-  }
-
-  // eslint-disable-next-line camelcase
-  UNSAFE_componentWillUpdate() {
-    this.componentWillUnmount();
-  }
-
-  componentDidUpdate() {
```

Comment:
Can this be safely deleted?

Reply:
I should be. Although it seems that we don't properly clean up the editor. I will check this again.

Reply:
I checked the code. Before this change `componentDidUpdate` would call `initQuill` that is guarded by `if (!this.editor) …` which is set during `componentDidMount`. So `componentDidUpdate` would always hit the guard and skip the contents of `initQuill`.